package.json

```diff
@@ -82,7 +90,8 @@
     "tether-drop": "^1.4.2",
     "tether-tooltip": "^1.2.0",
     "transliteration": "^1.1.9",
-    "url": "^0.11.0"
+    "url": "^0.11.0",
+    "webpack": "^1.14.0"
```

Comment:
I'm not sure but i think it's a dependency of a module we installed. I've removed it and will test